html-forms/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <form>
+      <div>
+        <label for="email">Email</label>
+        <br>
+        <input type="email" for="email" id="email">
+      </div>
+      <div>
+        <label for="whycode">Why is coding cool?</label>
+        <br>
+        <select name="whycode" id="whycode">
+          <option value="">Select an option</option>
+          <option value="Because it's interesting">Because it's interesting</option>
+          <option value="Because it's fun">Because it's fun</option>
+          <option value="Because it's powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <div>
+        Favorite Language (choose one)
+        <br>
+        <input type="radio" id="HTML" name="favorite" value="HTML">
+        <label for="favorite">HTML</label>
+        <input type="radio" id="CSS" name="favorite" value="CSS">
+        <label for="favorite">CSS</label>
+        <input type="radio" id="JavaScript" name="favorite" value="JavaScript">
+        <label for="favorite">JavaScript</label>
+      </div>
+      <div>
+        Would you like to sign up for more information?
+        <br>
+        <label for="signup">Sign me up!</label>
+        <input type="checkbox" id="signup" name="signup">
```

Comment:
You should have the `input` element before the label here, conventionally radio and checkbox inputs are followed by their label.